Sofa/Component/IO/Mesh/src/sofa/component/io/mesh/MeshOBJLoader.h

```diff
@@ -79,10 +79,8 @@ class SOFA_COMPONENT_IO_MESH_API MeshOBJLoader : public sofa::core::loader::Mesh
     virtual std::string type() { return "The format of this mesh is OBJ."; }
 };
 
-// deprecate usage of the lowcase version, but skip the warning when building SofaLoader
-#ifndef SOFA_BUILD_SOFALOADER
-using MeshObjLoader SOFA_ATTRIBUTE_DEPRECATED("v21.12 (PR#2428)", "v22.12", "MeshObjLoader has been renamed to MeshOBJLoader") = MeshOBJLoader;
-#endif // SOFA_BUILD_SOFALOADER
+SOFA_ATTRIBUTE_DISABLED("v21.12 (PR#2428)", "v22.12", "MeshObjLoader has been renamed to MeshOBJLoader")
+DeprecatedAndRemoved MeshObjLoader;
```

Comment:
It does not compile because of this, if two files are including this header, there will be twice a definition of MeshObjLoader  object.
In any case, this line (and some other) are weird: an alias (or a type) is being replaced by an instance of DeprecatedAndRemoved. So before MeshObjLoader is a type (or alias whatever) and after this is an object ?? 🤔